addons/test_website/tests/test_page_manager.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestWebsitePageManager(odoo.tests.HttpCase):
+    def test_page_manager_test_model(self):
+        if self.env['website'].search_count([]) == 1:
```

Comment:
Is this good practice to do that in tests? Should not you suppose website 2 exists? :grimacing:

Reply:
No because website 2 is from the demo data, and we run the testing suite without the demo data in the nightly (in prevision to one day replace the full testing suite by no-demo-data only).

Note that it's inspired from https://github.com/odoo/odoo/commit/842a981784f2b494d86258a28b7c603f887c88a5#diff-25e3dafb282af62ff4b4ce484718fcc58cd3bd749ee2ce7ee480d2e170e1b943R10